sdk/personalizer/azure-ai-personalizer/tests/test_configuration.py

```diff
@@ -0,0 +1,77 @@
+from devtools_testutils import AzureRecordedTestCase, recorded_by_proxy
```

Comment:
Sorry I didn't notice this before, but a lot of the files in tests/ are missing the Microsoft license header